js/react_native/yarn.lock

```diff
@@ -91,6 +144,19 @@
     "@babel/helper-replace-supers" "^7.16.7"
     "@babel/helper-split-export-declaration" "^7.16.7"
 
+"@babel/helper-create-class-features-plugin@^7.17.12", "@babel/helper-create-class-features-plugin@^7.18.0":
```

Comment:
why line 134 has same plugin but different version?

Reply:
I have cleaned up the lock file and regenerate a new one.